components/ModEditSubmissionForm.vue

```diff
@@ -1,41 +1,38 @@
 <template>
-    <div
+    <Modal
```

Comment:
@theyokohamalife In this case I think we should go further and instead of in this component only. The logic should be moved into the `Modal.vue` component since the logic is consistent always for showing it in any future component across the whole map. Or the `ModMainContent` in the same way it is used in the `MainContentContainer.vue` component as follows

```
  <Modal
                v-show="store.$state.isOpen"
                class="max-h-[calc(100vh-12rem)] ml-8 mt-12 overflow-y-auto"
            >
                <SearchResultDetails />
            </Modal>
```

Reply:
Sounds good! I can make the changes in a follow-up PR since it's a bit out of scope for this one.

Reply:
Sounds good. Would you like me to write another ticket for that? Or you got that one? @theyokohamalife

Reply:
I'll write up a ticket for it. 😎